File: app/src/main/java/com/neome/java/api/home/drawer/msg/MsgUserAvatarBulkGet.java
// neome.ai API - do not change
//

package com.neome.java.api.home.drawer.msg;

import com.neome.java.api.meta.base.Types.EntUserId;
import com.neome.java.api.nucleus.base.msg.Msg;

import java.util.Map;

@SuppressWarnings("unused")
public class MsgUserAvatarBulkGet extends Msg
{
  public Map<EntUserId, String> userIdVersionMap;
}

File: app/src/main/java/com/neome/api/home/drawer/msg/MsgUserAvatarBulkGet.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.drawer.msg

import com.neome.api.meta.base.Types.EntUserId
import com.neome.api.nucleus.base.msg.Msg
import java.util.Map

open class MsgUserAvatarBulkGet : Msg() {
    lateinit var userIdVersionMap: Map<EntUserId, String>
}
